content/en/user-guide/aws/qldb/index.md

```diff
@@ -8,13 +8,309 @@ aliases:
   - /aws/qldb/
 ---
 
-The Quantum Ledger Database (QLDB) API supports queries over cryptographically verifiable data, stored in a journal of immutable transaction events. LocalStack allows to create local ledgers and journals, to perform `CREATE TABLE` statements, to insert data via `INSERT` statements, and to query data via `SELECT` statements.
+## Introduction
 
-QLDB uses the [Amazon ION data format](https://amzn.github.io/ion-docs), a data serialization format that represents a superset of JSON, with a number of additional features.
+Quantum Ledger Database (QLDB) is supported by LocalStack only in the Pro version.
 
-A simple QLDB example running on LocalStack is provided in [this Github repository](https://github.com/localstack/localstack-pro-samples/tree/master/qldb-ledger-queries). The sample consists of two simple scenarios: (1) to create and list tables via the `pyqldb` Python library, and (2) to insert data into two tables and perform a `JOIN` query that combines data from the two tables. The sample output is posted below:
+Amazon Quantum Ledger Database is a fully managed ledger database service offered by Amazon Web 
+Services. It is designed to provide transparent, immutable, and cryptographically verifiable transaction 
+log functionality to applications. QLDB is particularly useful for applications that need a secure and scalable 
+way to maintain a complete and verifiable history of data changes over time. 
 
-```plaintext
+Some key features and aspects of AWS QLDB:
+
+**1. Ledger Database**: QLDB provides a ledger database, which is a centralized, immutable, and cryptographically verifiable journal of transactions.
+
+**2. Immutable and Append-Only**: Once data is written to the QLDB ledger, it is immutable. This means that transactions cannot be modified or deleted. New transactions are appended to the ledger.
+
+**3. Cryptographically Verifiable**: QLDB uses cryptographic hashing and digital signatures to ensure the integrity of data. Clients can verify the authenticity and integrity of the data in the ledger.
+
+**4. ACID Transactions**: QLDB supports ACID (Atomicity, Consistency, Isolation, Durability) transactions, ensuring that transactions are processed reliably and securely.
+
+**5. SQL-like Query Language**: QLDB supports PartiQL, which is a SQL-compatible query language. This allows developers familiar with SQL to query the data stored in the ledger.
+
+**6. Managed Service**: AWS QLDB is a fully managed service. AWS handles the operational aspects such as server provisioning, scaling, and maintenance, allowing developers to focus on building applications.
+
+**7. Integration with AWS Services**: QLDB can be integrated with other AWS services such as AWS Lambda, Amazon S3, and AWS Step Functions, enabling developers to create powerful and scalable applications.
```

Comment:
There is no need to have such a length background introduction. 3-4 lines about a service is enough